pkg/codegen/nodejs/gen.go

```diff
@@ -2415,7 +2415,7 @@ func genNPMPackageMetadata(pkg *schema.Package, info NodePackageInfo, localDepen
 		pluginVersion = version
 	}
 	// Parameterized schemas _always_ respect schema version
-	if pkg.SupportPack || pkg.Parameterization != nil {
+	if pkg.SupportPack {
```

Comment:
My only comment is perhaps we should rename this to something like

UseLanguageNamespacedSchemaVersion

or something of the like and then comment it to explain the old and new way and that Paramaterization requires the new way.

Reply:
Well it's more general than just using the schema version. This is also what changed where we write go.mods out.